src/components/home/Hero.jsx

```diff
@@ -1,5 +1,57 @@
+import React from 'react';
+
 const Hero = () => {
-  return <div>Hero</div>;
+  return (
+    <div style={styles.heroContainer}>
```

Comment:
remove this, use Tailwind styles
example: `<div className="flex flex-row items-center justify-between p-20 bg-[#fdf5e6]"></div>`

- no need to use flex-row after using flex, flex automatically orders everything into rows
- if you need to use a color that isn't in the tailwind.config.js file, add it to the list of colors
  - once you add it to the list of colors, it's a part of the theme, and you can call it using `bg-gardening-[the name you set]` (remember to remove those brackets)
- don't need to set a background color, the color is already set in the body